ConwaysGameOfLife/GameController.cs

```diff
@@ -70,43 +67,40 @@ public void Run()
             _output.WriteLine(Messages.Welcome);
             // create initial universe
             CreateInitialUniverse();

```

Comment:
We should prompt the user for input either here or first thing within `CreateInitialUniverse`
```suggestion
            // prompt
            CreateInitialUniverse();
```